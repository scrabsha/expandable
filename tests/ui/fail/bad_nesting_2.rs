#[expandable::expr]
#[allow(unused_macros)]
macro_rules! bad_nesting_example {
    ( $( $( $( $a:ident, )*, )+, )? ) => {
        $( $a )*
    };
}

fn main() {}
