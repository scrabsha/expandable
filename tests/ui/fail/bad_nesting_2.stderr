error: the matcher defines 3 repetitions (`?+*`) for a but the transcriber uses one repetition (`*`)
 --> tests/ui/fail/bad_nesting_2.rs:5:13
  |
5 |         $( $a )*
  |             ^
